docs/features/os.rst

```diff
@@ -84,3 +130,37 @@ your own argument parsing of the command line::
 
 Check the source code of this example, especially the ``main()`` function, to
 see the technique.
+
+Alternatively you can simply wrap the command plus arguments in quotes (either
+single or double quotes)::
+
+    $ python example/example.py "speak -p hello there"
+    ellohay heretay
+    (Cmd)
+
+Automating cmd2 apps from other CLI/CLU tools
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+While ``cmd2`` is designed to create **interactive** command-line applications
+which enter a Read-Evaluate-Print-Loop (REPL), there are a great many times
+when it would be useful to use a ``cmd2`` application as a run-and-done
+command-line utility for purposes of automation and scripting.
+
+This is easily achieved by combining the following capabilities of ``cmd2``:
+1. Ability to invoke a ``cmd2`` application with arguments
```

Comment:
Should these all be numbered 1?

Reply:
Great catch.  I had my Markdown vs reStructuredText wires crossed.  They should all be `#.` and there should be a blank line separating them from the rest of the text.